azerothcore/azerothcore-wotlk
# fix(DB/Creature): Add missing gossip option to obtain Yeh'kinya's Scr…

…oll. Source: TrinityCore.

Fixes #6707



## Issues Addressed:

- Closes #6707
- Closes https://github.com/chromiecraft/chromiecraft/issues/1070

## Tests Performed:

- Not tested

## How to Test the Changes:

1. `.quest add 3528`
2. `.quest complete 3528`
3. `.go creature 23270` (Yeh'kinya) and turn in
4. Talk to him again - there should be an option to get the new scroll


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
